tests/test_adaptor.py

```diff
@@ -174,6 +174,53 @@ def test_write_works(self):
 
         pin.write.assert_called_with(100)
 
+    def test_read_sets_up_pin(self):
+        self.edison.digital_read(5)
+
+        self.assertTrue(
```

Comment:
I've made the change here, and in a few other tests that would also benefit from using `assertIn`.